dnsmasq/files/dnsmasq.init

```diff
@@ -455,6 +460,8 @@ dhcp_add() {
 		DNS_SERVERS="$DNS_SERVERS $dnsserver"
 	}
 
+	dnsmasq_device_list="${dnsmasq_device_list} $ifname"
+	dnsmasq_interface_list="${dnsmasq_interface_list} $net"
```

Comment:
> This used after the dhcp_add, Line 1115 and 1124
You mean line 463 / 464 right ? This is where is put my comment, so I can see these lines, but I don't see this variable used in the dhcp_add function